luigi/server.py

```diff
@@ -214,7 +214,11 @@ def get(self, path):
 class RootPathHandler(tornado.web.RequestHandler):
 
     def get(self):
-        self.redirect("/static/visualiser/index.html")
+        visualization_graph = configuration.get_config().get('core', 'visualization_graph', '')
```

Comment:
Can you use the new syntax instead? `core().visualization_graph` I mean, and set the parameter to have a default of "svg" maybe?

On the other hand this is in the server code. Maybe it's good to not intermix with the client logic.